fec.config.js

```diff
@@ -13,4 +16,22 @@ module.exports = {
    * Add additional webpack plugins
    */
   plugins: [],
-};
\ No newline at end of file
+  moduleFederation: {
+    exposes: {
+      './RootApp': path.resolve(__dirname, './src/AppEntry.tsx'),
+      './DashboardWidget': path.resolve(
```

Comment:
Can we not name them all the same? Because there might be more widgets exposed from a single project.